### DISCUSSION TITLE
Getting stuck in Merge attempt failed

### DISCUSSION THREAD

**Discussion by TommyLeungZuto** [community/community#118526] [Pull Requests]
### Select Topic Area

Bug

### Body

I've tried multiple times still getting stuck. Any suggestion?

![image](https://github.com/community/community/assets/97601854/81620989-52e3-4eb0-ad6d-e5adf49ea185)

**Comment by eric-roberts-sp** [comment]
Also experiencing this.  I see this is closed, but what was the resolution?

**Reply by janwijbrand** [reply]
Yeah, I see this too. Would like to know the resolution as well...
